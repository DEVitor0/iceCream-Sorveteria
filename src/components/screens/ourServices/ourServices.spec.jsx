import { render } from '@testing-library/react';
import OurServices from './index';
import ImageContext from '../../../contexts/ImagesContext/ImageContext';
import '@testing-library/jest-dom';

const mockImages = {
  easeToBuy: 'mocked-easeToBuy-image',
  delivery: 'mocked-delivery-image',
  quality: 'mocked-quality-image',
};

describe('OurServices Component', () => {
  test('should match the snapshot', () => {
    const { asFragment } = render(
      <ImageContext.Provider value={mockImages}>
        <OurServices />
      </ImageContext.Provider>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
