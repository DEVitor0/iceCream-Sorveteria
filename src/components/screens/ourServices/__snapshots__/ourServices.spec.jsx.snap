// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurServices Component should match the snapshot 1`] = `
<DocumentFragment>
  <section
    class="services"
  >
    <div
      class="services__title"
    >
      <h2
        class="subtitle looking-services"
        id="services-subtitle"
      >
        Serviços
      </h2>
    </div>
    <div
      class="services__subtitle"
    >
      <h3
        class="title-services looking-services"
        id="services-title"
      >
        Sobre os nossos serviços
      </h3>
    </div>
    <div
      class="services__container looking-services"
      id="services-container"
    >
      <div
        class="kindOfServices-container"
      >
        <div
          class="kindOfServices-container__image"
        >
          <img
            class="kindOfServices-container__image__element"
            src="mocked-easeToBuy-image"
          />
        </div>
        <div
          class="kindOfServices-container__subheading"
        >
          <h4
            class="kindOfServices-container__subheading__element"
          >
            Fácil de pedir
          </h4>
        </div>
        <div
          class="kindOfServices-container__description"
        >
          <p
            class="paragraph "
            style="font-size: 20px;"
          >
            Peça sua sobremesa em segundos, sem complicação.
          </p>
        </div>
      </div>
      <div
        class="kindOfServices-container"
      >
        <div
          class="kindOfServices-container__image"
        >
          <img
            class="kindOfServices-container__image__element"
            src="mocked-delivery-image"
          />
        </div>
        <div
          class="kindOfServices-container__subheading"
        >
          <h4
            class="kindOfServices-container__subheading__element"
          >
            Entrega rápida
          </h4>
        </div>
        <div
          class="kindOfServices-container__description"
        >
          <p
            class="paragraph "
            style="font-size: 20px;"
          >
            Receba suas delícias com agilidade e segurança.
          </p>
        </div>
      </div>
      <div
        class="kindOfServices-container"
      >
        <div
          class="kindOfServices-container__image"
        >
          <img
            class="kindOfServices-container__image__element"
            src="mocked-quality-image"
          />
        </div>
        <div
          class="kindOfServices-container__subheading"
        >
          <h4
            class="kindOfServices-container__subheading__element"
          >
            Melhor qualidade
          </h4>
        </div>
        <div
          class="kindOfServices-container__description"
        >
          <p
            class="paragraph "
            style="font-size: 20px;"
          >
            Sabores premium feitos com os melhores ingredientes.
          </p>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
